### PR TITLE
VAT and National identification number for the fi_FI provider

### PR DESCRIPTION
**Pull request by lasselehtinen**

### CODE REVIEW

File: src/Faker/Provider/fi_FI/Payment.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+     * @see http://ec.europa.eu/taxation_customs/vies/faq.html?locale=en#item_11
+     * @see http://www.iecomputersystems.com/ordering/eu_vat_numbers.htm
+     * @see http://en.wikipedia.org/wiki/VAT_identification_number
+     * @see https://www.ytj.fi/en/index/vatnumber.html
+     *
+     * @return string VAT Number
+     */
+    public static function vat()
+    {
+        return sprintf("%s%d", 'FI', self::randomNumber(8, true));
```
Comment (fzaninotto) [inline src/Faker/Provider/fi_FI/Payment.php]: why not `sprintf('FI%d', static::randomNumber(8, true)` ?

File: src/Faker/Provider/fi_FI/Person.php [hunk 2]
```diff
@@ -85,4 +87,98 @@ class Person extends \Faker\Provider\Person
...
+            case '20':
+                $separator = 'A';
+                break;
+        }
+
+        return $separator;
+    }
+
+    /**
+     * Calculates the checksum for the National ID
```
Comment (fzaninotto) [inline src/Faker/Provider/fi_FI/Person.php]: any reference for the algorithm

**Comment by lasselehtinen** [comment]
I have done the fixes you suggested and also noticed that one of the docblocks was completely wrong. Can you give it another look?

**Comment by fzaninotto** [comment]
Good, but needs rebase

**Comment by lasselehtinen** [comment]
Hmm, I fetched the upstream master and fixed the merge conflicts in the readme. For some reason I cannot rebase the original pull request and the merge conflict fix commit together. It is visible in git log and everything but is not visible when I try to do the rebase. I tried my best to Google / SO this problem, but could not find anything. I will try to get this fixed, but any help is appreciated.

File: readme.md [hunk 3]
```diff
@@ -886,6 +886,24 @@ echo $faker->mobileNumber; // 082 123 5555
 echo $faker->dni; // '77446565E'
 ```
 
+### `Faker\Provider\fi_FI\Payment`
+=======
```
Comment (fzaninotto) [inline readme.md]: this one should be removed

**Comment by fzaninotto** [comment]
I don't mind the extra commit.

**Comment by lasselehtinen** [comment]
I fixed the readme, but I still cannot rebase this for some reason. If you want, I could ditch this PR and I would just create a new one from scratch.

Also for some reason Travis is failing to a test not really related to this PR at all.

**Comment by fzaninotto** [comment]
Yes, please recreate one from scratch. I'm closing this one.

**Pull request closed without merging**